### ISSUE TITLE
Web App for broader access to Moonlight

### ISSUE THREAD

**Issue by bayramog** [moonlight-stream/moonlight-qt#1090]
**Is your feature request related to a problem? Please describe.**
I want to use Moonlight on my Tesla vehicle. To do that I have only option as web browser. Since xCloud is working as web app, I can use but no Moonlight.

**Describe the solution you'd like**
Web based Moonlight client can increase devices we can utilize to enter Moonlight Streaming. So new development for Moonlight like Parsec.

**Describe alternatives you've considered**
Xcloud and Parsec has web apps. Xcloud is working but not for all games. Parsec has issue on web app with Tesla vehicles. 

**Additional context**

**Comment by Danik1601** [comment]
I am 100% certain that I saw ticket about PWA before, but I can't find it now.

TL:DR
They've been asked about it, but PWA (or RTC protocol specifically) currently do not support something they need.

**Comment by bayramog** [comment]
Yeah, I believe it cannot provide same super experience but being able to play my games on Tesla vehicle can be good option. I am using now Reemo as alternative solution. It is not great but acceptable.

**Comment by DanubeRS** [comment]
I too had this question, as I saw what [TestTube](testube.app) can do and that it supports a handful of other streaming platforms, including GeForce Now. No idea if the in-car network can support it well enough, but would be super cool!

**Comment by Argon2000** [comment]
Now possible with this unofficial Moonlight web stream fork that I've been working on for the past 6 months: https://github.com/Argon2000/moonlight-web-stream-tsla

Demo: https://youtu.be/whdvHChCQbg?si=zZ_1SllxLWe-VSH1

Requires both Sunshine and this middle-man software to stream to the Tesla browser. Works great for me over the cars built-in cellular network.
